### DISCUSSION TITLE
error[E0635]: unknown feature `proc_macro_span_shrink`

### DISCUSSION THREAD

**Discussion by rogertad** [b-inary/wasm-postflop#12] [Q&A]
Compiling object v0.31.1
   Compiling miniz_oxide v0.7.1
error[E0635]: unknown feature `proc_macro_span_shrink`
  --> C:\Users\roger\.cargo\registry\src\index.crates.io-6f17d22bba15001f\proc-macro2-1.0.56\src\lib.rs:92:30
   |
92 |     feature(proc_macro_span, proc_macro_span_shrink)
   |                              ^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0635`.
error: could not compile `proc-macro2` (lib) due to previous error

**Comment by b-inary** [comment]
Thank you for your report! I just updated the dependency, so please try again with the latest commit. If the problem persists, also try `cargo install wasm-pack` again.

**Comment by rogertad** [comment]
great! it is working now!!! cheers
